cmd/kubernetes-static/readme.md

```diff
@@ -4,23 +4,14 @@ Kubernetes Static is a project used to run the Kubernetes Integration locally on
 
 ## How it works
 
-The files in the `./data` folder are saved outputs from KSM and various kubelet endpoints, all required to run the full integration. 
+The files in the `./data` folder are saved outputs from KSM and various kubelet endpoints, all required to run the full integration.
 The program wil start a temporary HTTP server and serve these files. The groupers are configured to use these endpoints instead of discovering them.
```

Comment:
```suggestion
The files in the `./data` folder are saved outputs from KSM and various kubelet endpoints, which will be embedded at build-time and then served from a temporary HTTP. The groupers are configured to use these endpoints instead of discovering them.
```